### ISSUE TITLE
Movie Label Confusion

### ISSUE THREAD

**Issue by roshankern** [WayScience/mitocheck_data#4]
We noticed another minor issue in the labeled data sent by J-K Heriche (see https://github.com/WayScience/mitocheck_data/issues/2):

The morphological class names in `trainingset.dat` differ from the names specified in the [original Mitocheck paper](https://www.nature.com/articles/nature08869#Fig1). 

The table below documents the classes specified in the original paper vs `trainingset.dat`:
| Paper         | `trainingset.dat` |
| :-------------: | :-------------:|
| Interphase | Interphase |
| Elongated | Elongated |
| Large | Large |
| Prometaphase | Prometaphase |
| MAP | MetaphaseAlignment |
| Metaphase | Metaphase |
| Anaphase | ??? |
| Binuclear | ??? |
| Polylobed | ??? |
| Grape | Grape |
| Hole | Hole |
| Small Irr | SmallIrregular |
| Folded | Folded |
| Cell Death | Apoptosis |
| Condensed | UndefinedCondensed |
| Artefact | Artefact |

`trainingset.dat` also includes the following 3 morphological classes that I am unsure how to pair with the paper classes:

shape 1, shape 3,  ADCCM

**Comment by roshankern** [comment]
J-K kindly informed us that Shape1 and Shape3 correspond to binuclear and polylobed respectively. This was confirmed by visual analysis (see [example frame](https://github.com/WayScience/mitocheck_data/blob/main/3.extract_features/plateLT0017_19_well365_frame64.png)). 
Furthermore, ADCCM is a class grouping the other classes (ex A: artefact, D: dynamic/folded, C: condensed, etc). 
Thus, the anaphase class does not seem to be labeled in any of the training data.

**Comment by roshankern** [comment]
There are differences between the feature dataset sent by J-K and the manual annotations dataset described in ["automatic identification of chromosome morphologies"](https://www.sciencedirect.com/science/article/pii/S1047847709002731?via%3Dihub#aep-section-id21) that are causing confusion.

- Most notably, the `Anaphase` class does not exist in the J-K-sent data. This raises the question: do manual annotations for this class exist as indicated by the [automatic identification of chromosome phenotypes](https://www.sciencedirect.com/science/article/pii/S1047847709002731?via%3Dihub) paper? This category is obviously very important when classifying cell cycle.
- The `ADCCM` class exists in the J-K-sent data but not in the [automatic identification of chromosome phenotypes](https://www.sciencedirect.com/science/article/pii/S1047847709002731?via%3Dihub) paper. What is the point of this class?
- The instances of each label in the the J-K sent data differs from the instances mentioned in the [automatic identification of chromosome phenotypes](https://www.sciencedirect.com/science/article/pii/S1047847709002731?via%3Dihub) paper. In total about 9% of the labels are missing (ignoring the differences in ADCCM/anaphase).The instances of each label in the J-K-sent data are shown below:

| Label         | Instances |
| :-------------: | :-------------:|
| Interphase | 355 |
| Elongated | 34 |
| Large | 47 |
| Prometaphase | 363 |
| MAP | 196 |
| Metaphase | 75 |
| ADCCM | 105 |
| Binuclear | 291 |
| Polylobed | 478 |
| Grape | 97 |
| Hole | 128 |
| Small Irr | 182 |
| Folded | 51 |
| Cell Death | 170 |
| Condensed | 31 |
| Artefact | 86 |
| Total | 2689 |

The label instances as mentioned by the paper are shown in the confusion matrix for the model, with the total instances of the labels being 2957:

![matrix](https://ars.els-cdn.com/content/image/1-s2.0-S1047847709002731-gr3.jpg)
